### PR TITLE
Hypermode docs review changes

### PR DESCRIPTION
**Pull request by jacklynlee31**
➡️ [Writing down thoughts & making notes in this Notion doc.](https://www.notion.so/hypermodeinc/Docs-review-13ce0dd4f73680f5942cfd52a5625bfc?pvs=4)

---

Some comments/questions that are in the Notion doc but not captured in my changes:

✅ **Getting Started > Quickstart**
- Not sure how to get content to appear in **Inferences**. Do I need to build a model first? If so, we should mention that in this section, since the Observe function execution step makes it seem like all I had to do was run the query to get info on the Inferences page.
     - You do need to invoke the model from the function code first, so I added that into the docs just for extra clarification.

✅ **Develop > Work Locally**
~~- The `curl` command for installing Hyp CLI didn't work for me, but the `npm` command (on the Hyp CLI page) did work.~~
The `curl` command was fixed. See [this Slack thread](https://hypermode.slack.com/archives/C06LXM9GENA/p1732127649249269) for more details.

✅ **Develop > Configure Environment** & **Tools > Hosted Models**
- On the Configure Environment page, under Model hosting: "refer to the Hosted Models documentation for instructions on setting up dedicated models" - I'm only seeing references to shared models on that page, is there a difference between the two?

⏸️ **Manage > Modify Organization**
- `This forms the billing envelope for Hypermode.` - what does this mean?
- Comment in Notion doc: Means that Billing will be at the org level. Still thinking on this but I think it's okay to leave this as-is for now.

---

There are other questions/suggestions in the Notion file but I pulled the ones that I thought were higher priority into this PR.

### CODE REVIEW

File: create-project.mdx [hunk 1]
```diff
@@ -9,21 +9,21 @@ create a project through the Hypermode Console or Hyp CLI.
...
   project creation flow, you'll connect Hypermode to your GitHub account.
 </Note>
 
 From your organization home, click **New Project**. Set a name for the project
 and click **Create**.
 
-Once you have created a project, Hypermode prompts you to import the repo with
-your Modus app. To create your first Modus app, see the
-[Modus quickstart](modus/quickstart).
+Once you have created a project, Hypermode prompts you to finish setting up your
```
Comment (ryanfoxtyler) [inline create-project.mdx:19]: I wouldn't assume the user doesn't already have a Modus app here

File: introduction.mdx [hunk 2]
```diff
@@ -9,10 +9,10 @@ mode: "wide"
 ## What is Hypermode? {/* <!-- vale Google.Contractions = NO --> */}
 
 Hypermode is a managed service that provides the workbench and infrastructure to
-create **powerful, engaging, and secure AI assistants, APIs, and backend
-services**.
+create **powerful, engaging, and [secure](/security) AI assistants, APIs, and
```
Comment (ryanfoxtyler) [inline introduction.mdx:12]: I wouldn't link to the service's Security page here, as that doesn't cover the security architecture of the Modus app that is more directly being addressed here

File: work-locally.mdx [hunk 3]
```diff
@@ -3,28 +3,34 @@ title: Work Locally
...
-the [local Modus experience](/modus/modus-cli) with access to Hypermode-hosted
-models defined in your [app's manifest](/modus/app-manifest) for rapid
+By using the Hyp CLI, you can access Hypermode-hosted models defined in your
+[app's manifest](/modus/app-manifest). This allows for seamless and rapid
 experimentation with new AI models.
 
 ## Hyp CLI setup
 
-For access to hosted models, install the Hyp CLI to augment Modus.
+The Hyp CLI helps you manage your Hypermode projects. You can install the Hyp
```
Comment (ryanfoxtyler) [inline work-locally.mdx:12]: accessing models feels beyond managing projects

**Review by ryanfoxtyler — DISMISSED** [review]

File: create-project.mdx (hunk 1 above)
Comment (jacklynlee31) [reply create-project.mdx:19]: Agreed, I'll add that back in

File: introduction.mdx (hunk 2 above)
Comment (jacklynlee31) [reply introduction.mdx:12]: Okay! I'll remove the link

File: work-locally.mdx (hunk 3 above)
Comment (jacklynlee31) [reply work-locally.mdx:12]: I went back through this and I agree - let me adjust this

File: work-locally.mdx [hunk 4]
```diff
@@ -3,28 +3,35 @@ title: Work Locally
...
+By using the Hyp CLI, you can access Hypermode-hosted models defined in your
+[app's manifest](/modus/app-manifest). This allows for seamless and rapid
 experimentation with new AI models.
 
 ## Hyp CLI setup
 
-For access to hosted models, install the Hyp CLI to augment Modus.
+The Hyp CLI helps you access Hypermode-hosted models and manage your Hypermode
+projects. You can install the Hyp CLI to augment Modus and access your
+Hypermode-hosted models locally.
```
Comment (johnymontana) [inline work-locally.mdx:14]: I would be clearer that your models aren't running locally:

"and invoke your Hypermode-hosted models from your local Modus development environment"

**Review by johnymontana — APPROVED** [review]

**Review by ryanfoxtyler — APPROVED** [review]

**Pull request merged**